modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify the status of a boot source
+
+You can verify that a boot source is system-defined or user-defined.
+
+The `status` section of each boot source listed in the `status.dataImportChronTemplates` field of the `HyperConverged` CR indicates the type of boot source. For example, `commonTemplate: true` indicates a system-defined (`commonTemplate`) boot source. `status: {}` indicates a user-defined boot source.
```

Comment:
It looks a bit odd starting the last sentence with `status: {}`. Maybe consider combining the last two sentences:

```suggestion
The `status` section of each boot source listed in the `status.dataImportChronTemplates` field of the `HyperConverged` CR indicates the type of boot source. For example, `commonTemplate: true` indicates a system-defined (`commonTemplate`) boot source and `status: {}` indicates a user-defined boot source.
```

Reply:
Done.